recipes/polylaue/LICENSE

```diff
@@ -0,0 +1,39 @@
+Copyright © 2024, UChicago Argonne, LLC. All Rights Reserved
```

Comment:
The license file is in the tarball, no need to add it here again. See https://github.com/PolyLaue/PolyLaue/blob/0.0.1/LICENSE